misc/scripts/install-local.sh

```diff
@@ -59,6 +59,30 @@ function cget() {
   git ls-remote "$URL" "$BRANCH" | sed "s/refs\/heads\/.*//"
 }
 
+# logging metadata
+function loggingMeta() {
+	echo "_version=\"$version"\" | sudo tee "$LOGDIR"/"$PACKAGE" > /dev/null
+	echo "_description=\"$description"\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
+	echo "_date=\"$(date)"\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
+	if [[ $removescript == "yes" ]]; then
+	  echo "_removescript=\"yes"\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
+	fi
+	echo "_maintainer=\"$maintainer"\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
+}
+
+
+#Function to install .deb files
+function debpt() {
+    sudo apt install -f "$(echo "$url" | awk -F "/" '{print $NF}')"
```

Comment:
Install is something that is done in the install function of the pacscript so this would break that assumption.

Also the apt install wont work. Lets say `url="https://github.com/lbryio/lbry-desktop/releases/download/v0.51.1/LBRY_0.51.1.deb"` it will then try to run `sudo apt install -f LBRY_0.51.1.deb` from the repository. To make it install from a file you need to specify a path.

Reply:
apt also accepts relative paths

Reply:
We can also use pwd to solve this